INSTALL.md

```diff
@@ -108,23 +108,50 @@ The following contains scripts which will run all the required processes in orde
+                                   |        :7775    |                                     ||
+                                   |    +<-----------+                                     ||
+                                   |    |                                                  ||
+                                   | /sync    +--------------------------+                 ||
+                                   +--------->| dendrite-sync-api-server |<================++
+                                   |    |     +--------------------------+                 ||
+Matrix      +------------------+   |    |   :7773    |                                     ||
+Clients --->| client-api-proxy |---+    +<-----------+                                     ||
+            +------------------+   |    |                                                  ||
+          :8008                    | CS API   +----------------------------+               ||
+                                   +--------->| dendrite-client-api-server |==++           ||
+                                        |     +----------------------------+  ||           ||
+                                        |   :7771    |                        ||           ||
+                                        |            |            client_data ||           ||
+                                        +<-----------+         ++=============++           ||
+                                        |                      ||                          ||
+                                        |                      VV                          ||
+                                        |           +----------------------+    room_event ||
+                                        +---------->| dendrite-room-server |===============++
+                                        |           +----------------------+               ||
+                                        |         :7770                                    ||
+                                        |                      ++==========================++
+                                        +<------------+        ||
+                                        |             |        VV
+                                        |     +-----------------------------------+              Matrix
+                                        |     | dendrite-federation-sender-server |------------> Servers
+                                        |     +-----------------------------------+
+                                        |   :7776
+                                        |
+                                        +<-----------+
+                                                     |
+Matrix      +----------------------+  SS API  +--------------------------------+
+Servers --->| federation-api-proxy |--------->| dendrite-federation-api-server |
+            +----------------------+          +--------------------------------+
```

Comment:
Technically the federation-api-proxy talks to the media-api-server as well...

Reply:
Are you sure? how?

Reply:
Oh, I guess no one implemented that yet. It SHOULD talk to the media-api-server as well :)

Reply:
I'll fix that in a separate PR I guess.